packages/prisma-client/src/types.ts

```diff
@@ -6,5 +6,6 @@ export type {
   Project,
   Asset,
   DashboardProject,
+  AuthorizationRelation,
```

Comment:
what???

Reply:
the problem is that TS has to insert smth like `import('@webstudio-is/prisma-client').AuthorizationRelation` in the generated `.d.ts` for `@webstudio-is/authorization-token` but to do that the type has to be exported